validation_framework/deployer/engine_handler.py

```diff
@@ -31,29 +31,36 @@ def __init__(self,
                  include_peripherals: bool = False,
                  peripherals: list[str] = None):
         """
-        Engine handler constructor. It is in charge of assessing if the targets are passed as index or path to file.
-        If an index is passed has to find the corresponding file in the default folder, if it's a path, directly
-        loads the configuration.
+        Engine handler constructor. It is in charge of assessing if the targets are
+        passed as index or path to file. If an index is passed has to find the
+        corresponding file in the default folder, if it's a path, directly loads the
+        configuration.
         :param target_device: Index or path to target device file
-        :param nuvlaedge_version: Release version. Synchronized between NuvlaEdge and Deployment repository
+        :param nuvlaedge_version: Release version. Synchronized between NuvlaEdge and
+        Deployment repository
         """
         self.logger: logging.Logger = logging.getLogger(__name__)
         self.logger.debug(f'Creating Engine Handler for device {target_device}')
 
         # Assess device configuration
-        self.device_config_file: Path = target_device
-        if not self.device_config_file.is_file():
-            raise FileNotFoundError(f'Provided file {self.device_config_file} does not exists')
-
-        self.device_config: TargetDeviceConfig = utils.get_model_from_toml(TargetDeviceConfig, self.device_config_file)
-        self.device: SSHTarget = SSHTarget(self.device_config)
+        device_config_file: Path = target_device
+        if not device_config_file.is_file():
+            raise FileNotFoundError(f'Provided file {device_config_file} '
+                                    f'does not exists')
+
+        self.device_config: TargetDeviceConfig = utils.get_model_from_toml(
+            TargetDeviceConfig,
+            device_config_file)
+        self.coe: coe_base = coe_factory(self.device_config, nuvlaedge_version=nuvlaedge_version,
+                                         deployment_branch=deployment_branch,
+                                         nuvlaedge_branch=nuvlaedge_branch,
+                                         include_peripherals=include_peripherals,
+                                         peripherals=peripherals)
 
         self.engine_configuration: EngineEnvsConfiguration = EngineEnvsConfiguration()
-        self.release_handler = ReleaseHandler(self.device,
-                                              self.engine_configuration,
-                                              nuvlaedge_version,
-                                              deployment_branch,
-                                              nuvlaedge_branch)
+
+    def get_coe_type(self) -> str:
+        return self.coe.get_coe_type()
```

Comment:
This is fine but it's kind of standard procedure in python for getters to be properties of the class, i.e:

```python
@property
def coe_type(self) -> str:
    return self.coe.get_coe_type()
```
Then, they are called like an attribute